test/unit/assorted/uri_options.spec.test.ts

```diff
@@ -14,7 +14,10 @@ describe('URI option spec tests', function () {
     'tlsDisableCertificateRevocationCheck can be set to true',
     'tlsDisableCertificateRevocationCheck can be set to false',
     'tlsDisableOCSPEndpointCheck can be set to true',
-    'tlsDisableOCSPEndpointCheck can be set to false'
+    'tlsDisableOCSPEndpointCheck can be set to false',
+
+    // Skipped because we now only accept true/false for boolean options
```

Comment:
Maybe nice to note around here that the URI options spec expects warnings currently for invalid options and the Node driver throws.